### ISSUE TITLE
MOSIP-44902: PMS-Revamp: Keyboard navigation issue – Language dropdown not accessible on login page

### ISSUE THREAD

**Issue by Ivanmeneges** [mosip/partner-management-services#1772]
*Steps:*

1. Open the PMS Portal

2. Navigate to the login page.

3. Use keyboard functionality to navigate to the language selection option.

4. Attempt to open and select a language from the language option drop down.

*Actual Result:*
Keyboard focus skips the language dropdown initially and moves directly to username, password, login, and register options. The language selection is reached later, making it difficult to select the required language first.

![Log in to mosip - Google Chrome 2026-04-16 16-43-28.mp4](https://mosip.atlassian.net/browse/MOSIP-44902)

*Expected Result:*
Keyboard focus should first move to the language dropdown (top right), allowing the user to select the required language before proceeding with login.

### ISSUE
- **MOSIP-44515**
  Status: Closed

## 📎 Attachments

- [20260424-0750-54.1770769.mp4](https://mosip.atlassian.net/browse/MOSIP-44902)

- [Log in to mosip - Google Chrome 2026-04-16 16-43-28.mp4](https://mosip.atlassian.net/browse/MOSIP-44902)

**Comment by Ivanmeneges** [comment]
Rakshitha Sai Naraharisetti commented: In ALL browsers:

Page loads

 cursor is already in username

 typing works immediately 

{{autofocus}} forces cursor into username — tab logic works, but initial focus must be overridden

!20260424-0750-54.1770769.mp4|width=693,alt="20260424-0750-54.1770769.mp4"!
